promptsource/templates/swag/regular/templates.yaml

```diff
@@ -2,240 +2,130 @@ dataset: swag
+      \ instance %}\n{{answer_choices[0]}}\n{% else %}\n{{answer_choices[1]}}\n{%\
+      \ endif %}\n"
     metadata: !TemplateMetadata
       _do_eval: false
       _do_train: true
       choices_in_prompt: true
-      metrics: []
-      original_task: null
-    name: YesNo_0
-    reference: ''
+      metrics:
+      - Accuracy
+      original_task: false
+    name: appropriate_continuation
+    reference: The template checks if the two sentences are valid continuations.
   25b7abe7-e357-4e93-8c31-5f7be319b705: !Template
-    answer_choices: null
-    answer_choices_key: null
+    answer_choices:
+    - Ending 1
+    - Ending 2
+    - Ending 3
+    - Ending 4
+    answer_choices_key: ending0 ||| ending1 ||| ending2 ||| ending3
     id: 25b7abe7-e357-4e93-8c31-5f7be319b705
     jinja: "{{ startphrase }}...\nHow does the description likely end? \nEnding 1:\
       \ {{ ending0 }}\nEnding 2: {{ ending1 }}\nEnding 3: {{ ending2 }}\nEnding 4:\
-      \ {{ ending3 }}\n\n||| {{ {0: \"Ending 1\", 1: \"Ending 2\", 2: \"Ending 3\"\
-      , 3: \"Ending 4\"}[label] }}"
+      \ {{ ending3 }}\n||| \n{{ answer_choices[label] }}"
     metadata: !TemplateMetadata
       _do_eval: false
       _do_train: true
       choices_in_prompt: true
-      metrics: []
-      original_task: null
+      metrics:
+      - Accuracy
+      original_task: false
     name: how_ends
-    reference: ''
-  37348a67-d3ee-4c7b-a782-4cd532b32542: !Template
-    answer_choices: null
-    answer_choices_key: null
-    id: 37348a67-d3ee-4c7b-a782-4cd532b32542
-    jinja: "Consider the sentence: {{ sent2 }} {{ [ending0, ending1, ending2, ending3][3]\
-      \ }} \nIs it an appropriate continuation of the following situation:\n{{ sent1\
-      \ }} ?\nYes or No?\n||| {{ {0: \"No\", 1: \"No\", 2: \"No\", 3: \"Yes\"}[label]\
-      \ }}"
-    metadata: !TemplateMetadata
-      _do_eval: false
-      _do_train: true
-      choices_in_prompt: true
-      metrics: []
-      original_task: null
-    name: YesNo_reversed_3
-    reference: ''
-  5de009a6-014a-4859-8696-5ef8141b02d0: !Template
-    answer_choices: null
-    answer_choices_key: null
-    id: 5de009a6-014a-4859-8696-5ef8141b02d0
-    jinja: 'Consider the sentence: {{ sent1 }}
-
-      Is the following an appropriate continuation?
-
-      {{ sent2 }} {{ [ending0, ending1, ending2, ending3][2] }}
-
-      Yes or No?
-
-      ||| {{ {0: "No", 1: "No", 2: "Yes", 3: "No"}[label] }}'
-    metadata: !TemplateMetadata
-      _do_eval: false
-      _do_train: true
-      choices_in_prompt: true
-      metrics: []
-      original_task: null
-    name: YesNo_2
-    reference: ''
+    reference: 'Predict the ending key based on the startphrase. Not original task
+      because sent1 is not included. '
   66366555-f989-4e82-beca-2aaa92960a50: !Template
     answer_choices: null
-    answer_choices_key: null
+    answer_choices_key: ending0 ||| ending1 ||| ending2 ||| ending3
```

Comment:
Each of these needs to be wrapped in curly braces like `{{ending1}}`. Rewriting the output to use the `answer_choices` field will help catch these bugs.